.fernignore

```diff
@@ -1 +1,6 @@
 # Specify files that shouldn't be modified by Fern
+
+.github/workflows/publish.yml
+test/test_helper.rb
```

Comment:
What are these two test files?

Reply:
They are meant to be test stubs should people want to easily play around with the gem, they're programmatically generated and put into every ruby SDK in the latest version of the generator.
It also makes it easy to add test in the future should people want to